src/compiler/commandLineParser.ts

```diff
@@ -216,6 +216,13 @@ namespace ts {
             isCommandLineOnly: true,
             description: Diagnostics.Print_the_final_configuration_instead_of_building
         },
+        {
+            name: "listFilesOnly",
+            type: "boolean",
+            category: Diagnostics.Command_line_Options,

```

Comment:
@uniqueiniquity As far as I could tell, vstsc doesn't use this in build mode.  Does that sound right?

Reply:
Yes, it does. The idea is that in build mode, you should be depending on the incremental behavior of build mode itself, not our MSBuild version of it.

Reply:
To confirm, that's "Yes, it does sound right" and not "Yes, it does use the flag in build mode"?

Reply:
Yes, the former - it does sound right.